templates/neso

```diff
@@ -0,0 +1,10 @@
+
+cd $job_results
+$run_prefix
+
+
+echo "Running NESO Electrostatic2D3V"
+
+echo ${neso_base_dir}$neso_solver $neso_conditions_file $neso_mesh_file
+
+${neso_base_dir}$neso_solver $neso_conditions_file $neso_mesh_file
```

Comment:
neso_base_dir is defined in fabsim/deploy/machines_user.yml

This approach was recommended in the FabSIM documentation

Reply:
Shouldn't this be defined in a file local to this repository though as `fabsim/deploy/machines_user.yml` is outside of this repository? From the docs at https://fabsim3.readthedocs.io/en/latest/create_new_plugin/ it looks like we probably want a `machines_fabneso_user.yml` file in the root of the repository where any local environment variables are defined.

Reply:
I've updated the plugin to read from a local machines_FabNeso_user.yml file if such a file exists, and moved the base directory environment variable to there.

In the repository, I've included a machines_FabNeso_user_example.yml rather than the direct file, as this appears to be the FabSIM standard in the other plugins. FabNeso doesn't actually read this file, though, so the user has to change it locally. Happy to change this if you think the actual file would be better.